cypress/integration/signup.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Test for Signup', () => {
+  beforeEach(() => {
+    cy.visit('/signup');
+  });
+  it('Image should have alt as signup-icon', () => {
```

Comment:
@BukkyOmo Your test description should be readable and simple. For example, you could have 
```
it('should load an image element', () => {})
```
It is inside the test that you assert the attributes and other properties .